### ISSUE TITLE
Echopraxia

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#11036]
**Preferred term label:**

**Synonyms**
Mimicking of examiner's movements (Bush-Francis Catatonia Rating Scale)

**Definition (free text, please give PubMed ID)**
pathological repetition of the actions of other people as if echoing them (https://www.merriam-webster.com/dictionary/echo#medicalDictionary)

**Parent term (use hpo.jax.org/app)**
Atypical behavior HP:0000708

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Idiopathic catatonia ORPHA:648919

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@bcoleman96 
Melvin: Should we add a term called Echophenomenon to encompass echolalia, this, and  Palilalia, Echographia(?), Echoplasia ?

**Comment by pnrobinson** [comment]
@carolina-f 
Added as 
Echopraxia
HP:0020124
